CLI/test/CLITest.cs

```diff
@@ -477,22 +471,25 @@ internal static string Test(string workingDirectory, string arguments, ReturnCod
             System.Diagnostics.Process process = new System.Diagnostics.Process();
             System.Diagnostics.ProcessStartInfo startInfo = new System.Diagnostics.ProcessStartInfo();
             startInfo.WindowStyle = System.Diagnostics.ProcessWindowStyle.Hidden;
-            startInfo.FileName = "cmd.exe";
+            startInfo.FileName = "dotnet";
             startInfo.WorkingDirectory = workingDirectory;
-            // Find TypeCobol.CLI.exe location
+            // Find TypeCobol.CLI.dll location
             string currentDirectory = Environment.CurrentDirectory;
             string configuration = Path.GetFileName(currentDirectory);
             string pathToExe = Path.Combine(currentDirectory, "..", "..", "..", "..");
             pathToExe = Path.GetFullPath(pathToExe);
-            pathToExe = Path.Combine(pathToExe, "CLI", "src", "bin", configuration, "TypeCobol.CLI.exe");
-            startInfo.Arguments = @"/c " + pathToExe + " " + arguments;
+            pathToExe = Path.Combine(pathToExe, "CLI", "src", "bin", configuration, "TypeCobol.CLI.dll");
```

Comment:
Maybe we can rename `pathToExe `to `cliPath`?

Reply:
done